x/concentrated-liquidity/bench_test.go

```diff
@@ -0,0 +1,235 @@
+package concentrated_liquidity_test
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/osmosis-labs/osmosis/v15/app/apptesting"
+	clmodel "github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/model"
+	"github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types"
+)
+
+type BenchTestSuite struct {
+	apptesting.KeeperTestHelper
+}
+
+func (s BenchTestSuite) createPosition(accountIndex int, poolId uint64, coin0, coin1 sdk.Coin, lowerTick, upperTick int64) {
+	tokensDesired := sdk.NewCoins(coin0, coin1)
+
+	_, _, _, _, _, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(s.Ctx, poolId, s.TestAccs[accountIndex], tokensDesired, sdk.ZeroInt(), sdk.ZeroInt(), lowerTick, upperTick)
+	if err != nil {
+		// This can happen for ticks that map to the very small prices
+		// e.g 2 * 10^(-18) ends up mapping to the same sqrt price
+		fmt.Println("error creating position", err)
+	}
```

Comment:
Can we branch this off of https://github.com/osmosis-labs/osmosis/pull/5173 so we can remove this?

Reply:
This can't be removed because this would still lead to an error